components/form-toggle/test/index.js

```diff
@@ -17,15 +17,13 @@ describe( 'FormToggle', () => {
 			expect( formToggle.hasClass( 'components-form-toggle' ) ).to.be.true();
 			expect( formToggle.hasClass( 'is-checked' ) ).to.be.false();
 			expect( formToggle.type() ).to.equal( 'span' );
-			expect( formToggle.find( '.components-form-toggle__input' ).prop( 'value' ) ).to.be.undefined();
```

Comment:
could we replace `.prop( 'value' )` with `.prop( 'checked' )` instead of dropping the assertions?